### DISCUSSION TITLE
How mockoon identify changes in environment file.

### DISCUSSION THREAD

**Discussion by dharma412** [mockoon/mockoon#1449] [Help]
Scenario:

Step1:
I have imported Open API spec file and made chnages to one GET oprtation with custome response. On running via CLI i am able to get response .

Step2: while mockoon is running I have made one more change to same GET operation with new response , But mockoon is giving old response not the new response.

I observ one mockoon is not update new changes made in environment file.

Note: started mockoon via mockoon-cli starrt --data  file.json and hitting end point via Postman.

**Comment by 255kb** [comment]
The CLI does not support live updates, there is an [opened issue](https://github.com/mockoon/mockoon/issues/665).

**Reply by dharma412** [reply]
I have one more question does mockoon support updates in input specification file after server started. 

I have imported input swagger , hit few calls menawhile user has changed in few paratemeters in input swagger , does these changes update automatically or we need to import swagger again if so , how to bring the changes done in previous enviroment to latest swagger.

**Reply by 255kb** [reply]
Currently the OpenAPI/Swagger import can only be done once. It's on the roadmap to [allow reimporting](https://github.com/mockoon/mockoon/issues/765) and update a mock API. But I cannot give any ETA.
